src/app/Processor/ProcessorEdit/ProcessorEdit.tsx

```diff
@@ -136,75 +217,13 @@ const ProcessorEdit = (props: ProcessorEditProps): JSX.Element => {
     });
   }, [name, processorType, filters, transformation, action, source]);
 
-  const prepareFilters: (filters: EventFilter[]) => Partial<EventFilter>[] = (
```

Comment:
Moving `prepareFilters` and `prepareRequest` functions to `processorUtils`. 

They are used to convert the data coming from the processor form model into the payload for `ProcessorRequest`.